app/Http/Controllers/SongsController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Song;
+use DB;
+
+class SongsController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    public function index()
+    {
+        $songs = Song::orderBy('created_at', 'asc')->paginate(10);
+        return view('songs.index')->with('songs', $songs);
+    }
+
+    public function create()
+    {
+        return view('songs.add');
+    }
+
+    public function store(Request $request)
+    {
+        $this->validate($request, [
+            'title' => 'required',
+            'artist' => 'required',
+            'spotify_link' => 'required'
+        ]);
```

Comment:
De database bevat all constraints die aangeven dat title, artist en spotify_link niet null mogen zijn. Kan verder ook geen kwaad om extra te valideren of course.